companies/german-medical.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "german-medical",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Fairpreis GmbH",
+    "runs": [
+        "German Medical"
+    ],
+    "address": "Fairpreis GmbH\nKruppstraße 7\n47475 Kamp-Lintfort",
+    "phone": "+49 1573 3144515",
+    "email": "thiel@fairpreis-gmbh.de",
```

Comment:
Might be blocked by this: https://github.com/datenanfragen/data/pull/1133#issuecomment-913590834